packages/yourloops/test/integration/pages/menu/add-team-for-patient.spec.tsx

```diff
@@ -0,0 +1,191 @@
+    expect(screen.queryByRole('dialog')).not.toBeInTheDocument()
+  })
+  it('should close dialog policy privacy after clicking in cancel button', async () => {
+    await act(async () => {
+      renderPage('/')
+    })
+    mockGetTeamFromCod()
+
+    const badgeTeamMenu = screen.getByLabelText('Open team menu')
+    expect(badgeTeamMenu).toHaveTextContent('1')
+    await userEvent.click(badgeTeamMenu)
+    const buttonJoinTeam = screen.getByTestId('team-menu-teams-link')
+    await userEvent.click(buttonJoinTeam)
+    const dialog = screen.getByRole('dialog')
+    const titleDialog = within(dialog).getByTestId('team-add-dialog-title')
+    const labelDialog = within(dialog).getByTestId('label-dialog')
+    expect(dialog).toBeVisible()
+    expect(titleDialog).toHaveTextContent('Join the care team')
+    expect(labelDialog).toHaveTextContent('Please enter the team code (9 digits)')
+    const cancelButton = within(dialog).getByRole('button', { name: 'Cancel' })
+    const addTeamButton = within(dialog).getByRole('button', { name: 'Continue' })
+    const inputCode = within(dialog).getByRole('textbox')
+    expect(addTeamButton).toBeDisabled()
+    expect(cancelButton).toBeVisible()
+    await userEvent.type(inputCode, '263381988')
+    expect(addTeamButton).toBeEnabled()
+    await userEvent.click(addTeamButton)
+
+    const dialogPrivacy = screen.getByRole('dialog')
+    const addCareTeamButton = within(dialogPrivacy).getByRole('button', { name: 'Join team' })
+    const cancelCareTeamButton = within(dialogPrivacy).getByRole('button', { name: 'Cancel' })
+    const checkPolicy = within(dialogPrivacy).getByRole('checkbox')
+    const textPrivatePolicy = within(dialogPrivacy).getByTestId('text-privacy-policy')
+    const detailsTeam = within(dialogPrivacy).getByTestId('team-add-dialog-team-infos')
+
+    expect(within(dialogPrivacy).getByText('You are about to share you data with')).toBeVisible()
+    expect(detailsTeam).toHaveTextContent('fakeTeamFive6 rue des champs75000Paris263381988')
+    expect(within(dialogPrivacy).getByText('By accepting this invitation, I recognize this team as my care team and consent to share my personal data with all its members, who are authorized healthcare professionals registered on YourLoops. I acknowledge that I can revoke this access at any time.')).toBeVisible()
+    expect(dialogPrivacy).toHaveTextContent('Share your data with a care team')
+    expect(dialogPrivacy).toHaveTextContent('Please verify that the above details match the information provided by your healthcare professional before accepting the invitation.')
+    expect(textPrivatePolicy).toHaveTextContent('Read our Privacy Policy for more information')
+    expect(addCareTeamButton).toBeDisabled()
+    await userEvent.click(checkPolicy)
+    await userEvent.click(cancelCareTeamButton)
+    expect(screen.queryByRole('dialog')).not.toBeInTheDocument()
+  })
+
+  it('should after error in input code display an error message', async () => {
```

Comment:
```suggestion
  it('should display an error message after error in input code', async () => {
```
Again, this test could be done in the first one.
All these tests can be extracted in methods that could be defined in the `assert` folder